modules/pool/lib/usePoolUserTokenBalancesInWallet.tsx

```diff
@@ -11,8 +11,21 @@ export function _usePoolUserTokenBalancesInWallet() {
 
     const { userBalances, getUserBalance, ...userBalancesQuery } = useUserBalances(allTokenAddresses, allTokens);
 
-    const investTokens = pool.investConfig.options.map((option) => option.tokenOptions).flat();
-    const investableAmount = sumBy(investTokens, (token) =>
+    const tokenOptionsWithHighestValue = pool.investConfig.options.map((option) => {
+        const tokenWithHighestValue = orderBy(
+            option.tokenOptions,
+            (tokenOption) => priceForAmount({ ...tokenOption, amount: getUserBalanceForToken(tokenOption.address) }),
+            'desc',
+        )[0].address;
+
+        return {
+            address: tokenWithHighestValue,
+            amount: getUserBalanceForToken(tokenWithHighestValue),
+            hasMultipleTokenOptions: option.tokenOptions.length > 1,
+        };
+    });
+
+    const investableAmount = sumBy(tokenOptionsWithHighestValue, (token) =>
```

Comment:
investable amount is meant to include all tokens, not just the options with the highest value.